crates/iota-rest-api/src/system.rs

```diff
@@ -121,7 +121,11 @@ pub struct SystemStateSummary {
     /// during safe mode.
```

Comment:
one line above `rewards` should be `charges`

Reply:
This suggestion will be outdated due to https://github.com/iotaledger/iota/pull/5270/commits/3d65d4a39724a007ab66be516b08e4bb0fc44c9d and https://github.com/iotaledger/iota/pull/5270/commits/3f5933339b60e7b381332c09e578516965fc801c

`IotaSystemStateSummaryV1` will now be kept unchanged with field `safe_mode_computation_rewards` and `IotaSystemStateSummaryV2` introduced with renamed and additional fields.